<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayerWindow</class>
 <widget class="QMainWindow" name="PlayerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QToolButton" name="toolButton">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>160</y>
      <width>27</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>445</y>
      <width>410</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_moo">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>69</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_stop">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>433</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: solid;

background-color: rgba(191, 64, 64, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_next">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>433</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: solid;

background-color: rgba(191, 64, 64, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_like">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>433</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: solid;
background-color: rgba(191, 64, 64, 0);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_model">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>433</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: solid;

background-color: rgba(191, 64, 64, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_conv">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>433</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: solid;

background-color: rgba(191, 64, 64, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSlider">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>270</y>
      <width>22</width>
      <height>160</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>80</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>171</width>
      <height>361</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: solid;
background-color: rgba(191, 64, 64, 0);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_changedir">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>0</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: solid;

background-color: rgba(191, 64, 64, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>0</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="toolTipDuration">
     <number>-1</number>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: solid;

background-color: rgba(191, 64, 64, 0);</string>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <item>
     <property name="text">
      <string>0.5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>0.75</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1.0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1.5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2.0</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_systime">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>0</y>
      <width>147</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans CJK TC</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_mus_curtime">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>445</y>
      <width>41</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>0.00</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_mus_toltime">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>445</y>
      <width>41</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>0.00</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_ly">
    <property name="geometry">
     <rect>
      <x>195</x>
      <y>71</y>
      <width>421</width>
      <height>361</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: solid;
background-color: rgba(191, 64, 64, 0);
QListWidget::item
{
    color: white;
}
</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_prev">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>433</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: solid;

background-color: rgba(191, 64, 64, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_bk">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>480</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/pic/bk.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_changepic">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>0</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: solid;

background-color: rgba(191, 64, 64, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>41</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>girl</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>真爱粉</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>小黑子</string>
     </property>
    </item>
   </widget>
   <zorder>label_bk</zorder>
   <zorder>toolButton</zorder>
   <zorder>horizontalSlider</zorder>
   <zorder>label_moo</zorder>
   <zorder>pb_stop</zorder>
   <zorder>pb_next</zorder>
   <zorder>pb_like</zorder>
   <zorder>pb_model</zorder>
   <zorder>pb_conv</zorder>
   <zorder>verticalSlider</zorder>
   <zorder>listWidget</zorder>
   <zorder>pb_changedir</zorder>
   <zorder>comboBox</zorder>
   <zorder>label_systime</zorder>
   <zorder>label_mus_curtime</zorder>
   <zorder>label_mus_toltime</zorder>
   <zorder>listWidget_ly</zorder>
   <zorder>pb_prev</zorder>
   <zorder>pb_changepic</zorder>
   <zorder>comboBox_2</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
